History.md

```diff
@@ -1,5 +1,10 @@
 ## v.NEXT
 
+* Byte order marks included in `--settings` files will no longer crash the
+  Meteor Tool.
+  [Issue #5180](https://github.com/meteor/meteor/issues/5180)
+  [PR #9459](https://github.com/meteor/meteor/pull/9459)
```

Comment:
Can we move this bullet item a little further down the list, since it's somewhat in descending order of notability (no offense to these changes!)? 😉